### ISSUE TITLE
dmc.Notification broken

### ISSUE THREAD

**Issue by subsurfaceiodev** [snehilvj/dash-mantine-components#231]
Hi!
Using latest version == 0.12.1 we found a bug for dmc.Notification. Using the following mre:

```
import dash_mantine_components as dmc
from dash_iconify import DashIconify
from dash import Dash, callback, Input, Output, html

app = Dash(__name__)

app.layout = dmc.NotificationsProvider(
    html.Div(
        [
            html.Div(id='notifications-container'),
            dmc.Button('Show Notification', id='notify'),
            dmc.Tabs(
                [
                    dmc.TabsList(
                        [
                            dmc.Tab('Gallery', value='gallery'),
                            dmc.Tab('Messages', value='messages'),
                            dmc.Tab('Settings', value='settings'),
                        ]
                    ),
                    dmc.TabsPanel('Gallery tab content', value='gallery'),
                    dmc.TabsPanel('Messages tab content', value='messages'),
                    dmc.TabsPanel('Settings tab content', value='settings'),
                ],
                color='red',
                orientation='vertical',
            )
        ]
    )
)


@callback(
    Output('notifications-container', 'children'),
    Input('notify', 'n_clicks'),
    prevent_initial_call=True,
)
def show(n_clicks):
    return dmc.Notification(
        title='Hey there!',
        id='simple-notify',
        action='show',
        message='Notifications in Dash, Awesome!',
        icon=DashIconify(icon='ic:round-celebration'),
    )


if __name__ == '__main__':
    app.run_server()
```

We noticed that notification keeps appering when user interacts with other unassociated elements that should not trigger it. See following video:

https://github.com/snehilvj/dash-mantine-components/assets/109676812/26e2bfd9-05a1-4e8e-8b97-ebecc872c303

**Comment by subsurfaceiodev** [comment]
Using latest alpha version == 0.13.0a3  wont even allow running mre, raises the following: AttributeError: module 'dash_mantine_components' has no attribute 'NotificationsProvider'

**Comment by subsurfaceiodev** [comment]
Nevermind, the issue was caused by dash==2.16.0, upgrading to dash==2.16.1 solved main issue, but alpha version == 0.13.0a3 issue persists
